### ISSUE TITLE
feature: advanced playlist search

### ISSUE THREAD

**Issue by jessicajonesunitedkingdom** [emoon/HippoPlayer#114]
Hi

If you plan to do this: an advanced search window.
Most players do not have good search.

I mean advanced search in the current playlist: searching substring in song's title, song length, file format, search inside metadata, ... and also combinations of this like: "find all songs with "demo" inside song's title and are XM format" or "find all songs of format DRO"
A search results windows could pop up (separate from playlist) in which you can double click the found files to play them.

**Comment by emoon** [comment]
I'm currently storing all metadata in a local sqlite database so things like this should be quite easy to do.

**Comment by jessicajonesunitedkingdom** [comment]
LOL, I just wanted to ask for SQLite :-D
You can use SQL directly on the database.

**Comment by emoon** [comment]
Yeah I just need to build a query statement so should be simple to do.
